review/communication-react.api.md

```diff
@@ -0,0 +1,2130 @@
+
+// @public
+export type CallCompositeOptions = {
+    mobileView?: boolean;
+    errorBar?: boolean;
+    callControls?: boolean | CallControlOptions;
+};
+
+// @public
+export type CallCompositePage = 'accessDeniedTeamsMeeting' | 'call' | 'configuration' | 'joinCallFailedDueToNoNetwork' | 'leftCall' | 'lobby' | 'removedFromCall';
+
+// @public
+export interface CallCompositeProps extends BaseCompositeProps<CallCompositeIcons> {
+    adapter: CallAdapter;
+    // (undocumented)
+    callInvitationUrl?: string;
+    options?: CallCompositeOptions;
+}
+
+// @public
+export interface CallCompositeStrings {
+    cameraLabel: string;
+    cameraPermissionDenied: string;
+    cameraTurnedOff: string;
+    close: string;
+    complianceBannerNowOnlyRecording: string;
+    complianceBannerNowOnlyTranscription: string;
+    complianceBannerRecordingAndTranscriptionSaved: string;
+    complianceBannerRecordingAndTranscriptionStarted: string;
+    complianceBannerRecordingAndTranscriptionStopped: string;
+    complianceBannerRecordingSaving: string;
+    complianceBannerRecordingStarted: string;
+    complianceBannerRecordingStopped: string;
+    complianceBannerTrancriptionStarted: string;
+    complianceBannerTranscriptionConsent: string;
+    complianceBannerTranscriptionSaving: string;
+    complianceBannerTranscriptionStopped: string;
+    configurationPageCallDetails?: string;
+    configurationPageTitle: string;
+    defaultPlaceHolder: string;
+    failedToJoinCallDueToNoNetworkMoreDetails?: string;
+    failedToJoinCallDueToNoNetworkTitle: string;
+    failedToJoinTeamsMeetingReasonAccessDeniedMoreDetails?: string;
+    failedToJoinTeamsMeetingReasonAccessDeniedTitle: string;
+    learnMore: string;
+    leftCallMoreDetails?: string;
+    leftCallTitle: string;
+    lobbyScreenConnectingToCallMoreDetails?: string;
+    lobbyScreenConnectingToCallTitle: string;
+    lobbyScreenWaitingToBeAdmittedMoreDetails?: string;
+    lobbyScreenWaitingToBeAdmittedTitle: string;
+    microphonePermissionDenied: string;
+    mutedMessage: string;
+    networkReconnectMoreDetails: string;
+    networkReconnectTitle: string;
+    privacyPolicy: string;
+    removedFromCallMoreDetails?: string;
+    removedFromCallTitle: string;
+    soundLabel: string;
+    startCallButtonLabel: string;
+}
+
+// @public
+export type CallControlOptions = {
+    compressedMode?: boolean;
```

Comment:
`compressed` is a weird word that refers to file compression algorithms usually.
In UI design this is usually called `compact`, can we rename that?

Example: MS Teams
![image](https://user-images.githubusercontent.com/1885005/141979250-9c84966f-b6be-43d3-90ab-573a34efdc61.png)

Reply:
Should we avoid the boolean and do `formFactor: 'comfy' | 'compact'`? (maybe default instead of comfy)

Reply:
PR for this change: https://github.com/Azure/communication-ui-library/pull/1096

Reply:
@JamesBurnside "formFactor" is mobile or desktop, here it's about UI density.
I see that your PR settled on `displayType` which is okay - type is a little generic, have we considered `displayDensity`?